Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1294,7 +1294,7 @@ class BrowserViewController: UIViewController {
         if let homePanelController = homePanelController {
             return .HomePanels(homePanelState: homePanelController.homePanelState)
         }
-        guard let tab = tabManager.selectedTab where !tab.loading else {
+        guard let tab = tabManager.selectedTab else {
             return .Loading
```

Comment:
Ah, so `webView.loading` should definitely not be `.Loading`.

Carry on. 👍 

Moving `.isPrivate` to an accessor should be done in a separate bug.